xbmc/cores/dvdplayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -439,6 +439,19 @@ bool CDVDDemuxFFmpeg::Open(CDVDInputStream* pInput, bool streaminfo, bool filein
 
   if (m_streaminfo)
   {
+    if (CSettings::GetInstance().GetBool(CSettings::SETTING_AUDIOOUTPUT_SUPPORTSDTSHDCPUDECODING))
```

Comment:
Summary:

Reasons for this setting:
- Users have 2.0 output only and don't care at all if dts-hd 7.1 is decoded and downmixed afterwards
- With the setting we can keep it disabled by default, non hifi users won't realize anything
- Low performance devices (on android, windows (old dual core machines)) can enable it on a per file basis (*)

* Not automatically detectable without high error margin